components/feedback_modal/feedback.scss

```diff
@@ -15,4 +15,9 @@
         font-size: 15px;
         resize: none;
     }
+
+    &__FreeFormTextLimit {
+        color: red;
```

Comment:
Prefer `var(--dnd-indicator)`, that's what we use elsewhere to indicate error/danger